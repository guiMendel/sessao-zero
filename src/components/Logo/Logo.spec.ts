import { mountOptions } from '@/tests'
import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import { Logo } from '.'

describe('Logo', () => {
  it('should render properly', () => {
    const wrapper = mount(Logo, mountOptions)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
