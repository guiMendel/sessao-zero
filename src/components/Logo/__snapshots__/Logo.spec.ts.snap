// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logo > should render properly 1`] = `"<h1 data-v-4bfea8c3=\\"\\">SessÃ£o Zero</h1>"`;
